src/ray/stats/stats.h

```diff
@@ -132,7 +132,7 @@ static inline void InitOpenTelemetryExporter(const int metrics_agent_port,
           absl::ToInt64Milliseconds(StatsConfig::instance().GetReportInterval())),
       /*timeout=*/
       std::chrono::milliseconds(
-          absl::ToInt64Milliseconds(StatsConfig::instance().GetHarvestInterval())));
+          absl::ToInt64Milliseconds(0.9 * StatsConfig::instance().GetReportInterval())));
```

Comment:
See if we can remove 0.9

Reply:
Could you leave a comment saying why we do that? Also I feel `absl::ToInt64Milliseconds(StatsConfig::instance().GetReportInterval())) - 1` is better than `0.9`.

Reply:
agree ^^

this also seems to indicate that setting the timeout very close to the interval might be an antipattern from OT's perspective